app/services/WhatYouOweService.scala

```diff
@@ -44,23 +44,25 @@ class WhatYouOweService @Inject()(val financialDetailsService: FinancialDetailsS
     (chargeItem.transactionType, chargeItem.subTransactionType) match {
       case (_, Some(Nics2)) => true
       case (PoaOneDebit | PoaTwoDebit | PoaOneReconciliationDebit
-            | PoaTwoReconciliationDebit | BalancingCharge | MfaDebitCharge, _) => true
+            | PoaTwoReconciliationDebit | BalancingCharge | LateSubmissionPenalty
```

Comment:
for sure worth to move all these in a list, so to easy up reading pattern matching: 
`.. FirstLatePaymentPenalty| SecondLatePaymentPenalty| MfaDebitCharge ... `